### PR TITLE
Allow InitLog operation on PREORDERED_LOG

### PR DESCRIPTION
**Pull request by pavelkalinnikov**

### CODE REVIEW

File: server/log_rpc_server.go [hunk 1]
```diff
@@ -37,6 +37,7 @@ const proofMaxBitLen = 64
 
 var (
 	optsLogRead            = trees.NewGetOpts(trees.Query, true, trillian.TreeType_LOG, trillian.TreeType_PREORDERED_LOG)
+	optsLogInit            = trees.NewGetOpts(trees.Queue, false, trillian.TreeType_LOG, trillian.TreeType_PREORDERED_LOG)
```
Comment (Martin2112) [inline server/log_rpc_server.go]: trees.Admin perhaps?

Comment (pavelkalinnikov) [reply server/log_rpc_server.go]: Yup, also fixed `optsPreorderedLogWrite` while I'm here.
I wonder if there is any difference between them apart from naming, seems to work both ways.

**Comment by Martin2112** [comment]
Can you also make the admin change to map_rpc_server if required?

These operation types aren't checked at the moment but they will be soon.

**Comment by pavelkalinnikov** [comment]
@Martin2112 done.

**Pull request merged**